fsd_utils/simple_utils/date_utils.py

```diff
@@ -1,13 +1,27 @@
 from datetime import datetime
 
+import pytz
+
 
 def current_datetime_after_given_iso_string(value: str) -> bool:
-    today = datetime.today().now()
+    # Grab local timezone
+    uk_tz = pytz.timezone("Europe/London")
+    # Grab current localised datetime
+    now_datetime = datetime.now(uk_tz)
+    # Convert now_datetime to offset-naive
+    now_datetime_naive = now_datetime.replace(tzinfo=None)
+    # Convert value datetime to correct format for camparison
     parsed = datetime.fromisoformat(value)
-    return today > parsed
+    return now_datetime_naive > parsed
 
 
 def current_datetime_before_given_iso_string(value: str) -> bool:
-    today = datetime.today().now()
+    # Grab local timezone
+    uk_tz = pytz.timezone("Europe/London")
+    # Grab current localised datetime
+    now_datetime = datetime.now(uk_tz)
+    # Convert now_datetime to offset-naive
+    now_datetime_naive = now_datetime.replace(tzinfo=None)
+    # Convert value datetime to correct format for camparison
     parsed = datetime.fromisoformat(value)
-    return today < parsed
+    return now_datetime_naive < parsed
```

Comment:
```suggestion
    return now_datetime_native < parsed
```

Reply:
the naive is for timezone naive, it is unclear so let me change the naming for this variable!